.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+#####################################################
+#
+# List of approvers for OpenTelemetry .NET Instrumentation
+#
+#####################################################
+#
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/master/community-membership.md
+# 
+#
+# Learn about CODEOWNERS file format: 
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @open-telemetry/dotnet-approvers @macrogreg 
```

Comment:
What is the reasoning for making all of the .NET SDK approves here?
I think some of the folks do not have the right context.

Reply:
I think this must be a temporary thing until separate Github group is created. It should be a separate group from sdk repo with one or two folks in each other group.

Reply:
Yeah, my idea was to have a broader list on CODEOWNERS and let the rights on the repo control who could merge PRs. However, per doc [here]():

Reply:
My ignorance in this regard: the teams were already created, see https://github.com/orgs/open-telemetry/teams?query=dotnet-instrumentation

Only maintainers have the right to merge PRs: at this moment that will be me, @macrogreg and @cijothomas.